From be3ecff8b1bc1de4d29ca6381ee9ab42e6c15947 Mon Sep 17 00:00:00 2001
From: Mingli Yu <mingli.yu@windriver.com>
Date: Fri, 12 Nov 2021 15:33:53 +0800
Subject: [PATCH] tabfiles-tags: add use-system-commands option

Pass --use-system-commands option to make blkid command available
Fixes:
 # ./ts/libmount/tabfiles-tags
 [snip]
 ./ts/libmount/../../functions.sh: line 652: /usr/lib64/util-linux/ptest/blkid: No such file or directory
 FAILED (libmount/tabfiles-tags)'

Upstream-Status: Inappropriate [configuration specific]

Signed-off-by: Mingli Yu <mingli.yu@windriver.com>
---
 tests/ts/libmount/tabfiles-tags | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/ts/libmount/tabfiles-tags b/tests/ts/libmount/tabfiles-tags
index 69b06a9..17176ef 100755
--- a/tests/ts/libmount/tabfiles-tags
+++ b/tests/ts/libmount/tabfiles-tags
@@ -4,7 +4,7 @@ TS_TOPDIR="${0%/*}/../.."
 TS_DESC="tags"
 
 . $TS_TOPDIR/functions.sh
-ts_init "$*"
+ts_init "$* --use-system-commands"
 
 ts_skip_nonroot
 
-- 
2.17.1

